config/webpacker.yml

```diff
@@ -83,7 +83,7 @@ test:
   compile: true
 
   # Compile test packs to a separate directory
-  public_output_path: packs-test
+  #public_output_path: packs-test
```

Comment:
I think this will cause problems for the developers because any compilation in the test environment will end up in the 'normal' webpacker output path.

From yesterdays comments in Slack I don't think this should be needed anymore?

Reply:
Yes, we can now revert this since we are building the assets in `production` env.

Reply:
I've reverted this change. This should keep the test environment unaffected, and assets are still built fine with the changes we made yesterday.